### ISSUE TITLE
Crash on any input in --debug mode

### ISSUE THREAD

**Issue by nurmukhametov** [ispc/ispc#2711]
On any input with `--stdlib` and `--debug`, ISPC crashes:

```bash
$  ispc --debug t.ispc -o t.ll -O2

stdlib.ispc:2146:20: Error: Unable to find any matching overload for call to function "atomic_swap_global".
        Passed types: (/*unbound*/ int64 * uniform, varying int64)

/localdisk2/anurmukh/applications.compilers.ispc.core/src/main.cpp(404): FATAL ERROR: Unhandled signal sent to process; terminating.
***
```

**Comment by nurmukhametov** [comment]
The smaller reproducer is:
```ispc
void foo(void *uniform dst, void *uniform src, uniform int32 count) {
    __memcpy32((int8 * uniform) dst, (int8 * uniform) src, count);
}
```
to reproduce:
```bash
$ ispc t.ispc -o t.o -O2 --target=avx2-i32x8 --debug --nostdlib
Initial AST
`-Function <t2.ispc:2.5 - 2.67>  "foo"
  `-(body) StmtList <t2.ispc:2.5 - 2.67>
    `-ExprStmt <t2.ispc:2.5 - 2.66>
t2.ispc:2:5: Error: Unable to find any matching overload for call to function "__memcpy32" only considering exact matches.
        Passed types: (/*unbound*/ int8 * uniform, /*unbound*/ int8 * uniform, uniform int32)

    __memcpy32((int8 * uniform) dst, (int8 * uniform) src, count);
    ^^^^^^^^^^
```

**Comment by pbrubaker** [comment]
Is this fixed now by #2716 ?
